### DISCUSSION TITLE
Best way to import Joplin notes?

### DISCUSSION THREAD

**Discussion by nextcloudfan** [TriliumNext/Trilium#4798] [Q&A]
Hi,

I exported my Joplin notes in the format MD+ Front matter (only Markdown will not export the source of the note, i.e. weblinks the notes is clipped from, etc).

Then imported them into Trillium. But notes, which are clipped to Joplin with the Joplin webclipper and which contain pictures and look perfectly fine in Joplin, are now missing this pictures after the import into Trillium.

Can anyone help or as any ideas?

Thanks

**Comment by Nriver** [comment]
You can use trilium-py to import joplin markdowns. https://github.com/Nriver/trilium-py?tab=readme-ov-file#import-from-joplin It will handle the pictures for you.

As for the front matter, I've never tried it.
